src/main/java/io/confluent/connect/jdbc/DataConverter.java

```diff
@@ -322,7 +323,12 @@ private static void convertFieldValue(ResultSet resultSet, int col, int colType,
 
       case Types.NUMERIC:
       case Types.DECIMAL: {
-        colValue = resultSet.getBigDecimal(col);
+        int scale = resultSet.getMetaData().getScale(col);
```

Comment:
Can we test this change in JdbcSourceTaskConversionTest where we test a bunch of other conversions?